src/renderer/store/modules/utils.js

```diff
@@ -359,12 +359,21 @@ const actions = {
       }
 
       case 'search': {
-        if (!url.searchParams.has('search_query')) {
+        let searchQuery = null
+        if (url.searchParams.has('search_query')) {
+          // https://www.youtube.com/results?search_query={QUERY}
+          searchQuery = url.searchParams.get('search_query')
+        }
+        if (url.searchParams.has('q')) {
+          // https://redirect.invidious.io/search?q={QUERY}
+          searchQuery = url.searchParams.get('q')
+        }
+        if (searchQuery == null) {
           throw new Error('Search: "search_query" field not found')
         }
 
-        const searchQuery = url.searchParams.get('search_query')
         url.searchParams.delete('search_query')
+        url.searchParams.delete('q')
```

Comment:
I wonder if the `delete` should be moved into the `if`s

Reply:
I agree, moving it to the ifs is a good idea, we don't need to run it if we know that the parameter doesn't exist.